README.md

```diff
@@ -93,6 +93,7 @@ Defaults to `latest`.
   - If set to `true`, will post comment every time.
   - If set to `nonzero`, will post comment only if any checks failed or there's changes to the Terraform plan ([returncode](#outputs) other than 0).
 - `github_token`: (optional) Github access token, required to post PR comments.
+- `issue_number`: (optional) If set, post comment to a specific issue or PR instead of the current one.
```

Comment:
Should we name this `pr_number` instead of `issue_number`?

Technically it is issue_number cause github regards pr and issue as the same, but from the user point of view, pr_number might be better, since posting terraform check to an issue doesn't make a lot of sense?

Unless nobita has a use case of posting terraform check message to issue?

Reply:
Actually in nobita, we're planning to post it to an auto-generated issue when preparing for production releases.

But personally, I don't mind if it's `pr_number` or `issue_number`. I just matched what GitHub named the parameter in their API 😅

Reply:
Naruhodo. In that case `issue_number` does make sense 👍